src/cli/bin.js

```diff
@@ -2,6 +2,7 @@
 
 'use strict'
 
+require('pipe-args').load()
```

Comment:
Cool module!

It is doing a bit more than what we want, though. If enabled, every command will take stdin->args, which opens space for other errors.